src/mocks/reference-data/mock-reasons-for-abandoning.reference-data.ts

```diff
@@ -0,0 +1,81 @@
+import { ReasonsForAbandoning, ReferenceDataResourceType } from '@models/reference-data.model';
+import { VehicleTypes } from '@models/vehicle-tech-record.model';
+
+export const mockReasonsForAbandoning: ReasonsForAbandoning[] = [
+  {
```

Comment:
Need to pull out the reasons for abandoning which are relevant to other test types and implement some sort of filtering based on the vehicle type

Reply:
OR instead of filtering, there could be a way of having the `resoureType` field as "PsvReasonsForAbandoning" && "HgvReasonsForAbandoning" ... and some sort of system which determines which reasons for abandoning to ask the reference data endpoint for

Reply:
Don't think this is in scope of this story.

Reply:
Maybe, maybe not